test/v2/custom-widget.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<script>grtpAPI = '/lib/v2/'</script>
+<script>gratipayURI = '/test/gratipay.com/'</script>
+
+<body>
+    <div data-gratipay-teamslug="gratipay" data-gratipay-widget="custom">
+        Username: <span class="gratipay-slug"></span>
+        <a class="gratipay-profile-link">profile link</a><br>
+        Identity: <span class="gratipay-identity"></span><br>
```

Comment:
We can drop identity, too. We're just dealing with Teams now, which are all plural.